apps/docs/blog/2023-02-04-announcing-rxangular-cdk-and-template-1-0.md

```diff
@@ -0,0 +1,150 @@
+
+The main way to use RenderStrategies is through template directives and pipes, but the library also provides the `RxStrategyProvider` service for component-level interaction.
+This service offers the `schedule` and `scheduleWith` methods for scheduling any type of work, with `schedule` returning an observable and `scheduleWith` returning a MonoTypeOperatorFunction for use in an rxjs pipe.
+
+The `scheduleCD` method is an imperative option for scheduling change detection cycles.
+
+Try the RxStrategyProvider and see rendering strategies in action in this [stackblitz demo](https://stackblitz.com/edit/angular-ivy-1vfpoe?file=src%2Fapp%2Fapp.component.ts).
+
+#### [Zone configuration](https://www.rx-angular.io/docs/cdk/zone-configurations) and [Zone-less utils:](https://www.rx-angular.io/docs/cdk/zone-less) Reliable Zone.js tuning
+
+Our CDK provides zone-configuration utils for partially disabling zone.js, reducing the risk of unexpected bugs.
+These utils provide autocompletion, event groups, and assertions to ensure proper usage.
+The zone-less utils allow for selective disabling of zone patching for specific browser APIs, including requestAnimationFrame, cancelAnimationFrame, setInterval, clearInterval, setTimeout, clearTimeout.
+The utility function `getZoneUnpatchedApi` can be used to get an unpatched version of any browser API.
+
+#### [Coalescing:](https://www.rx-angular.io/docs/cdk/coalescing) Optimize Frequent Events
+
+Coalescing sub-package provides set of utilities that implement a technique of merging multiple emissions, streams, or calls into one during a given time frame to improve the performance.
+This helps to minimize the number of updates made to the user interface, improving the overall efficiency and speed of the application.
+This technique is one of the key building blocks for optimizing change detection cycles in Angular, and is utilized to ensure that the application is functioning smoothly and effectively.
+
+#### [Coercing:](https://www.rx-angular.io/docs/cdk/coercing) Seamless Type Conversion
+
+Our coercion package provides a set of utilities to perform type coercion (data type conversion) in a seamless and efficient manner.
+These utilities include factories, such as 'coerceObservable' and 'coerceDistinctObservable', and operators, such as 'coerceObservableWith' and 'coerceDistinctWith'.
+These tools help to ensure that data is in the desired format before it is used in the application, improving overall functionality and preventing unexpected errors.
+
+#### [Transformations](https://www.rx-angular.io/docs/cdk/api/transformation-helpers) Tree Shakable Entity Management
+
+The transformation helpers provided by the `@rx-angular/cdk/transformations` are a complete set of utilities to manage entities. They are immutable by default and include transformation functions to handle insertions, updates, dictionary <-> array conversions & more to help you build store like state management systems.
```

Comment:
```suggestion
The transformation helpers provided by the `@rx-angular/cdk/transformations` are a complete set of utilities to manage entities. They are immutable by default and include transformation functions to handle insertions, updates, dictionary <-> array conversions & more to help you build store like state management systems.


## Roadmap

We also want to give a brief overview about the future steps we want to take with the RxAngular libraries.

Right now, the RxAngular monorepo and all of its libraries are based on angular v12. This is because of backwards compatibility to pre-ivy projects.
While we like the idea of being backwards compatible, we have made the decision to move forward and upgrade the whole project to the latest angular version available. We want to make sure to build modern solutions based on the new APIs introduced with the latest changes to the angular framework itself.

Some things we are investigating right now:

* new provider functions (`withFeature`)
* easier creation of state (`inject`)
* signals integration into `@rx-angular/state`
* signals integration into `@rx-angular/template`
* concurrent mode with signals

That being said, we will release the next major versions very soon and we have decided to sync the `@rx-angular/*` versions with the angular versions, giving us the ability to provide two breaking changes a year.

| RxAngular  | Angular version  |
|------------|------------------|
| `1.0.0`    | `>= 12`          |
| `2.0.0`    | `>= 13`          |
| `14.0.0`   | `^14`            |
| `15.0.0`   | `^15`            |
| `16.x`     | `^16`            |


```